### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SC7639**
Added a  Get Reactide Running (While in beta) section to the README.md. I had trouble getting things started and I've seen so many issues and comments on issues about getting Reactide running so I thought this would be useful at least for now

### CODE REVIEW

**Comment by okbrown** [comment]
Why hasn't this been merged yet? It is very useful information to have, unless someone is going to provide a better way of starting this project?

**Comment by SC7639** [comment]
I need someone with write access to merge the pull request please

**Comment by oshalygin** [comment]
@SC7639 I wouldn't hold your breath.  I had nearly the same PR last week and they closed it with this reasoning:

_hey there, we aren't including this information in the readme as Reactide is still a ways from production. We're working hard to get a beta up with installers asap. Cheers!_

https://github.com/reactide/reactide/pull/60

**Comment by SC7639** [comment]
@oshalygin ahh, well if they close it then that's up to them, thanks for letting me know.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ Reactide is a cross-platform desktop application that offers a custom simulator,
   <img alt="Reactide Screenshot" src="http://reactide.io.s3-website-us-west-1.amazonaws.com/images/reactide-screenshot2.png">
 </p>
 
+## Get Reactide Running (While in beta)
+Frist you need to run ``npm install`` once finished you will need two terminal (OSX/Linux) or cmd/powershell (Windows) instances. In the first run ```npm run dev``` and in the second run ```npm run dev-start```, a window should open and you can start using Reactide.
```
Comment (zacbraddy) [inline README.md]: Typo on "First"

Comment (SC7639) [inline README.md]: Thanks @zacbraddy I've corrected the typo

**Comment by oshalygin** [comment]
image

@SC7639 sigh.

**Comment by SC7639** [comment]
@oshalygin I'm not surprised after what you said happened with yours. There needs to a part in the readme at least that says that an installer or executable is coming soon and to stay peeled and watch out for it. Most people will be put off by the difficulty in starting the project

**Pull request closed without merging**